### PR TITLE
Add position to plot interactive and others

### PR DESCRIPTION
**Pull request by zhut19**
Adding the position of S2 peaks as well as the main S2 position in the interactive plot
Here for s1 peak, orange X mark the main s2 position, while for other s2 peaks orange X still makrs the main s2 position but a red X mark the position of the peak you are looking at.

s1_000000_074776
s2_000000_074776

A few other minor changes is done in this pr as well
1) Add rise time to peak information
2) Remove marking 'unknown' peaks
3) Comment out some configs
4) Show neural net position

### CODE REVIEW

File: pax/plugins/plotting/Plotting.py [hunk 1]
```diff
@@ -849,22 +869,21 @@ def draw_peak(self):
...
-            pos = peak.get_position_from_preferred_algorithm(['PosRecTopPatternFit', 'PosRecNeuralNet',
+            pos = peak.get_position_from_preferred_algorithm(['PosRecNeuralNet', 'PosRecTopPatternFit', 
                                                               'PosRecRobustWeightedMean', 'PosRecWeightedSum',
                                                               'PosRecMaxPMT'])
         except ValueError:
             peak_text += "Position reconstruction failed!"
         else:
-            peak_text += 'Top hitpattern reconstruction: (%0.2f, %0.2f), gof %0.1f.\n' % (
-                pos.x, pos.y, pos.goodness_of_fit)
+            peak_text += 'Neural network reconstruction: (%0.2f, %0.2f), gof %0.1f.\n' % (pos.x, pos.y, pos.goodness_of_fit)
```
Comment (tunnell) [inline pax/plugins/plotting/Plotting.py]: __[Flake8]__

[E501] line too long (124 > 120 characters)

File: pax/plugins/plotting/Plotting.py [hunk 2]
```diff
@@ -849,22 +869,21 @@ def draw_peak(self):
...
         peak_text += 'Fraction in top: %0.2f\n' % peak.area_fraction_top
-        peak_text += 'Peak widths: hit time std = %dns,\n' \
-                     ' 50%% area range = %dns, 90%% area range = %dns\n' % (peak.hit_time_std,
+        peak_text += 'Peak shape: rise time = %dns,\n' \
+                     ' 50%% area range = %dns, 90%% area range = %dns\n' % (-peak.area_decile_from_midpoint[1],
                                                                             peak.range_area_decile[5],
                                                                             peak.range_area_decile[9])
         try:
-            pos = peak.get_position_from_preferred_algorithm(['PosRecTopPatternFit', 'PosRecNeuralNet',
+            pos = peak.get_position_from_preferred_algorithm(['PosRecNeuralNet', 'PosRecTopPatternFit',
```
Comment (tunnell) [inline pax/plugins/plotting/Plotting.py]: __[Flake8]__

[W291] trailing whitespace

File: pax/plugins/plotting/Plotting.py [hunk 3]
```diff
@@ -718,8 +739,7 @@ def plot_event(self, event):
             epoch_to_human_time(self.trigger_time_ns),
             self.trigger_time_ns % units.s)
         suspicious_channels = np.where(event.is_channel_suspicious)[0]
-        event_text += 'Suspicious channels (# hits rejected):\n ' + ', '.join([
-            '%s (%s)' % (ch, event.n_hits_rejected[ch]) for ch in suspicious_channels]) + '\n'
+
```
Comment (tunnell) [inline pax/plugins/plotting/Plotting.py]: __[Flake8]__

[W293] blank line contains whitespace

File: pax/plugins/plotting/Plotting.py [hunk 4]
```diff
@@ -849,22 +869,22 @@ def draw_peak(self):
...
+            pos = peak.get_position_from_preferred_algorithm(['PosRecNeuralNet', 'PosRecTopPatternFit',
                                                               'PosRecRobustWeightedMean', 'PosRecWeightedSum',
                                                               'PosRecMaxPMT'])
         except ValueError:
             peak_text += "Position reconstruction failed!"
         else:
-            peak_text += 'Top hitpattern reconstruction: (%0.2f, %0.2f), gof %0.1f.\n' % (
-                pos.x, pos.y, pos.goodness_of_fit)
+            peak_text += 'Neural network reconstruction: (%0.2f, %0.2f), gof %0.1f.\n'\
+            % (pos.x, pos.y, pos.goodness_of_fit)
```
Comment (tunnell) [inline pax/plugins/plotting/Plotting.py]: __[Flake8]__

[E122] continuation line missing indentation or outdented

File: pax/plugins/plotting/Plotting.py [hunk 5]
```diff
@@ -849,21 +869,21 @@ def draw_peak(self):
...
         try:
-            pos = peak.get_position_from_preferred_algorithm(['PosRecTopPatternFit', 'PosRecNeuralNet',
+            pos = peak.get_position_from_preferred_algorithm(['PosRecNeuralNet', 'PosRecTopPatternFit',
                                                               'PosRecRobustWeightedMean', 'PosRecWeightedSum',
                                                               'PosRecMaxPMT'])
         except ValueError:
             peak_text += "Position reconstruction failed!"
         else:
-            peak_text += 'Top hitpattern reconstruction: (%0.2f, %0.2f), gof %0.1f.\n' % (
+            peak_text += 'Neural network reconstruction: (%0.2f, %0.2f), gof %0.1f.\n'% (
```
Comment (tunnell) [inline pax/plugins/plotting/Plotting.py]: __[Flake8]__

[E225] missing whitespace around operator

**Comment by zhut19** [comment]
@pdeperio The lines removed from waveform plot like 'veto', 'busy' lines are actually in XENON1T.ini config, so one don't need to look into code but simply uncomment those in the config to draw them again.

For 'unknown' peaks, they are still marked in the plot but no longer have text 'unknown' next to them

I did hard coded in the Plotting to remove printing out the suspicious channels, as far as I know, they don't really help us understand the peak. And sometimes run out of space to print and cover some of the peak parameters.

**Comment by pdeperio** [comment]
Manually copied to ```pax_v6.8.0``` environment on Midway:
```
/project/lgrandi/anaconda3/envs/pax_v6.8.0/lib/python3.4/site-packages/pax-6.8.0-py3.4.egg/pax
```

**Pull request merged**